tactics/class_tactics.ml

```diff
@@ -37,7 +37,7 @@ module NamedDecl = Context.Named.Declaration
 
 (** Options handling *)
 
-let typeclasses_debug = ref 0
+let typeclasses_debug = ref 1
```

Comment:
What is this change?

Reply:
I just enabled to see the debug messages as it tested in CI. I will remove it once CI passed.

Reply:
OK